radio/src/storage/sdcard_yaml.cpp

```diff
@@ -264,9 +264,9 @@ const char * writeModelYaml(const char* filename)
 // EEPROM slot simulation based on file names:
 // - /MODELS/model[00-99].yml
 
-constexpr uint8_t MODELIDX_STRLEN = sizeof(MODEL_FILENAME_PREFIX "00");
+uint8_t MODELIDX_STRLEN = sizeof(MODEL_FILENAME_PREFIX "00");
```

Comment:
Please use “const”, so that lands in flash and not RAM.

Reply:
Or even better, move the “constexpr” into header. It is then equivalent to a macro.

Reply:
Hm... I could swear I tried that at the time, and was getting 'out of scope' warnings but it seems to be compiling now.